sdk/python/examples/psycopg2_opentelemetry.py

```diff
@@ -0,0 +1,39 @@
+"""This is an example that shows Approzium Opentelemetry integration. It also
+integrates with a Jaeger service to export and view generated traces.
+"""
+from opentelemetry import trace
+from opentelemetry.ext import jaeger
+from opentelemetry.ext.psycopg2 import Psycopg2Instrumentor
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchExportSpanProcessor
+
+import approzium
+import approzium.opentelemetry
+from approzium.psycopg2 import connect
+
+auth = approzium.AuthClient("authenticator:6001")
+approzium.default_auth_client = auth
+
+trace.set_tracer_provider(TracerProvider())
+
+jaeger_exporter = jaeger.JaegerSpanExporter(
+    service_name="approzium_service", agent_host_name="jaeger", agent_port=6831
```

Comment:
I'm thinking of naming the server side of the trace, `"approzium_authentication_server"`. Do you think we should name this, `"approzium_authentication_client"`?

Or I'm totally open to changing the name used server side. If you'd like to propose a set of client/server names, go for it!

Reply:
Oh. The thing is, this is usually set by the user if they want to use Jaeger exporter. In other words, even if they are not using Approzium, they would still have to pick a service name for their service. I named it approzium_service just for the example.

Reply:
👍 that makes sense!